packages/context/src/oidcContext/AuthenticationContext.provider.tsx

```diff
@@ -165,15 +165,10 @@ export const AuthenticationProviderInt = ({
   );
 };
 
-type AuthenticationProviderProps = Omit<AuthenticationProviderIntProps,
-  | 'authenticationServiceInt'
-  | 'CallbackInt'
-  | 'setLoggerInt'
-  | 'OidcRoutesInt'
-  | 'oidcLogInt'
-  | 'authenticateUserInt'
-  | 'logoutUserInt'
->
+type AuthenticationProviderProps = Omit<
+  AuthenticationProviderIntProps,
+  'authenticationServiceInt' | 'CallbackInt' | 'setLoggerInt' | 'OidcRoutesInt' | 'oidcLogInt' | 'authenticateUserInt' | 'logoutUserInt'
```

Comment:
**suggestion:** you could use an `enum` here

Reply:
For a Type ?